Update merge process in docs (#493)
MicrosoftDocs/WindowsCommunityToolkitDocs: .github/PULL_REQUEST_TEMPLATE.md

<!-- When opening a PR, start by forking this repository. Then, based on the type of change you're making you'll need to create a new branch from either the `master` or `dev` branches:

> If you have a typo or existing document improvement to an already shipped feature, please base your change off of the [Master branch](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/tree/master). This will allow us to get the change to the published documentation between releases.

Reviewer: ```suggestion
If you have a typo or existing document improvement to an already shipped feature, please base your change off of the [master branch](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/tree/master). This will allow us to get the change to the published documentation between releases.
```

For documentation regarding any new features, please base your fork off the last updated dev branch. For example: 'dev/7.1.0'.

> We will merge updates from the current dev branch to 'master' to keep master in-sync with the live branch. When we make a new release, we will push the live branch in order to publish documentation for new features.

Reviewer: ```suggestion
We will merge updates from the current main branch to the latest dev branch to keep it in-sync with any updated changes. We will periodically sync the main branch automatically with the live branch to pick up any changes made over time. When we make a new release, we will integrate the dev branch into the main branch in order to publish documentation for new features.
```

Please check if your PR fulfills the following requirements:

> - [ ] Correctly picked the right branch to base the change off (`dev` for new features, `Master` for typos/improvements)

Reviewer: ```suggestion
- [ ] Correctly picked the right branch to base the change off (`dev` for new features, `master` for typos/improvements)
```
